lib/scenic/statements.rb

```diff
@@ -60,7 +69,7 @@ def create_view(name, version: nil, sql_definition: nil, materialized: false)
     # @example Drop a view, rolling back to version 3 on rollback
     #   drop_view(:users_who_recently_logged_in, revert_to_version: 3)
     #
-    def drop_view(name, revert_to_version: nil, materialized: false)
+    def drop_view(name, revert_to_version: nil, defined_as: nil, materialized: false)
```

Comment:
This isn't used in this method?

Reply:
They handle migration reversals elsewhere and mirror arguments between the migration and rollback code. (That's also where the `revert_to_version` business comes from.) Didn't want to disrupt the standing pattern.